tests/integration/test_issues.py

```diff
@@ -130,3 +137,42 @@ def test_issue_1304_arc_sketch_creation():
     finally:
         # Reverse the default units to meter
         DEFAULT_UNITS.LENGTH = UNITS.meter
+
+
+def test_issue_1309_revolve_operation_with_coincident_origins(modeler: Modeler):
+    """Test that revolving a sketch with coincident origins (sketch and rotation origin)
+    does not crash the program.
+
+    For more info see
+    https://github.com/ansys/pyansys-geometry/issues/1309
+    """
+    # Sketch Plane
+    sketch_plane = Plane(
+        origin=Point3D([0, 0, 5]), direction_x=UNITVECTOR3D_X, direction_y=UNITVECTOR3D_Z
+    )
+
+    # Create Base
+    sketch = Sketch(sketch_plane)
+
+    sketch.arc(
+        start=Point2D([0, 0]),
+        end=Point2D([4.7, 4.7]),
+        center=Point2D([0, 4.7]),
+        clockwise=False,
+    ).segment_to_point(Point2D([4.7, 12])).segment_to_point(Point2D([-25.3, 12])).segment_to_point(
+        Point2D([-25.3, 0])
+    ).segment_to_point(Point2D([0, 0]))
```

Comment:
This part is a bit hard to read IMO, I would do something like this if it works:
```suggestion
    arc_test = sketch.arc(
        start=Point2D([0, 0]),
        end=Point2D([4.7, 4.7]),
        center=Point2D([0, 4.7]),
        clockwise=False,
    )
    arc_test = arc_test.segment_to_point(Point2D([4.7, 12]))
    arc_test = arc_test.segment_to_point(Point2D([-25.3, 12]))
    arc_test = arc_test.segment_to_point(
        Point2D([-25.3, 0])
    )
    arc_test = arc_test.segment_to_point(Point2D([0, 0]))
```

Reply:
Ah I see what the problem is. I forgot to add the parenthesis so that it is properly formatted. Let me fix this.

Reply:
Fixed in https://github.com/ansys/pyansys-geometry/pull/1336/commits/38b4cd8a96964d986923150192c4d1b32dac45d8